Formula/twilio@rc.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class TwilioATrc < Formula
+  desc "unleash the power of Twilio from your command prompt"
+  homepage "https://github.com/twilio/twilio-cli"
+  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v2.27.0-rc.1/twilio-v2.27.0-rc.1.tar.gz"
```

Comment:
Is there a versioning beyond patch as well? 
`-rc.1` 

Might be missing something here, but shouldn't the versioning be more like:
`v2.27.0-rc`
`v2.27.1-rc` ...

Reply:
This is the same convention used for CLI as well. https://github.com/shamantraghav/twilio-cli/releases